doc/user_guide/dynamo0p3.rst

```diff
@@ -1934,6 +1925,59 @@ As described in the PSy-layer :ref:`Argument Intents
 :ref:`field <dynamo0.3-field>` objects is always ``in``. The field or
 scalar that has its data modified by a Built-in is marked in **bold**.
 
+.. _dynamo0.3-api-built-ins-metadata:
+
+Metadata
+++++++++
+
+The code below outlines the elements of the LFRic API Built-in
+metadata, 1) 'meta_args', 2) 'iterates_over' and 3) 'procedure'::
+
+  type, public, extends(kernel_type) :: aX_plus_bY
+     private
+     type(arg_type) :: meta_args(5) = (/                              &
+          arg_type(GH_FIELD, GH_WRITE, ANY_SPACE_1),                  &
+          arg_type(GH_REAL,  GH_READ              ),                  &
+          arg_type(GH_FIELD, GH_READ,  ANY_SPACE_1),                  &
+          arg_type(GH_REAL,  GH_READ              ),                  &
+          arg_type(GH_FIELD, GH_READ,  ANY_SPACE_1)                   &
+          /)
+     integer :: iterates_over = DOFS
+   contains
+     procedure, nopass :: aX_plus_bY_code
+  end type aX_plus_bY
+
+As can be seen, the Built-ins metadata are the same as the
+:ref:`user-defined Kernel metadata <dynamo0.3-api-kernel-metadata>`
+with one exception and that is having ``DOFS`` instead of ``cells`` as
+the iteration space.
+
+.. _dynamo0.3-built-ins-valid-access:
+
+Valid Access Modes
+##################
+
+The allowed accesses for arguments in Built-in
+kernels are a bit different than for the
+:ref:`user-defined Kernels <dynamo0.3-kernel-valid-access>` and
+are listed in the table below.
+
+.. tabularcolumns:: |l|l|l|
+
++---------------+----------------+---------------------------------+
+| Argument Type | Function Space | Access Type                     |
++===============+================+=================================+
+| GH_INTEGER    | n/a            | GH_READ, *GH_SUM*               |
```

Comment:
Not your fault but since we're here, please could you remove GH_SUM from this row since we mark it in italics and then say actually, it's not valid!

Reply:
Removed. I left the note below as it is.